### PR TITLE
Instructions for accessing Thoth's public database on opf-datacatalog

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to https://github.com/thoth-station/datasets/issues/32

## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] No

## This Pull Request implements

Instructions in docs to access the public Thoth database that will be published on `opf-datacatalog`.

### CODE REVIEW

**Comment by mayaCostantini** [comment]
/hold
Waiting for database review and publication

**Review by pacospace — APPROVED** [review]
💯 Thanks!! 🚀

File: README.rst [hunk 1]
```diff
@@ -446,3 +446,19 @@ After initialization, you are ready to retrieve data
 
     except NotFoundError:
         # File does not exist
+
+Accessing Thoth Data on the Operate-First Public Bucket
+=======================================================
+
+A public instance of Thoth's database is available on the `Operate-First Public Bucket
+<https://github.com/operate-first/apps/blob/4008fc6117314a6b689f4ffa9264d423db57a99f/docs/content/odh/trino/access_public_bucket.md>`__ for external contributors to start developing components of Thoth.
```
Comment (goern) [inline README.rst:454]: should this reference the master branch rather than a specific commit?

Comment (mayaCostantini) [reply README.rst:454]: Right, thank you!

Comment (fridex) [reply README.rst:454]: It might be better to reference instructions in the `datasets` repository to maintain instructions only in one place.

Comment (mayaCostantini) [reply README.rst]: Good idea, I will reference a link to the corresponding section in `datasets` once https://github.com/thoth-station/datasets/pull/67/files is merged :+1:

Comment (mayaCostantini) [reply README.rst]: Changed by https://github.com/thoth-station/storages/pull/2565/commits/f271abc82697b18fdf1ddd1cfed1660fa1f76c86

**Pull request merged**